frontend/src/component/project/Project/Project.styles.ts

```diff
@@ -1,8 +1,12 @@
 import { styled, Tab } from '@mui/material';
 import { FavoriteIconButton } from 'component/common/FavoriteIconButton/FavoriteIconButton';
 
-export const StyledDiv = styled('div')(() => ({
+export const StyledDiv = styled('div')(({ theme }) => ({
     display: 'flex',
+    maxWidth: '90%',
+    [theme.breakpoints.down('md')]: {
+        maxWidth: '70%',
+    },
```

Comment:
Where do you get the percentages from? Have you manually checked with the other elements? Wouldn't it be easier to make the top row a flex container and set its width to 100%? Flex grow the middle and voila?

Reply:
that's what we had before and text-ellipsis didn't work. I think it always adds the ellipsis when there's width or max-width

Reply:
I tested this visually to make sure it looks good on all screen sizes and found by trial and error that on larger devices 90% is fine while on smaller ones it's 70%